.github/workflows/aws-lambda-java-runtime-interface-client.yml

```diff
@@ -32,18 +32,18 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v2.5.0
 
     - name: Set up JDK 1.8
-      uses: actions/setup-java@v1
+      uses: actions/setup-java@v3.7.0
       with:
         java-version: 1.8
 
     - name: Set up QEMU
-      uses: docker/setup-qemu-action@v1
+      uses: docker/setup-qemu-action@v2.1.0
 
     - name: Set up Docker Buildx
-      uses: docker/setup-buildx-action@v1
+      uses: docker/setup-buildx-action@v2.2.1
```

Comment:
Suggest simplifying to `@v2` for both of these, and elsewhere where you've specified a more granular semver